util/__tests__/assignClientContext.test.mjs

```diff
@@ -0,0 +1,46 @@
+import { assignClientContext } from '@/util/assignClientContext';
+
+const mockContext = {
+  frontmatter: { title: 'Sample Title' },
+  pathname: '/sample-path',
+  headings: ['Heading 1', 'Heading 2'],
+  readingTime: {
+    text: '2 mins read',
+    minutes: 2,
+    time: 120000,
+    words: 200,
+  },
+  filename: 'sample-file.md',
+};
+
+describe('assignClientContext', () => {
+  it('should assign properties to the client context', () => {
+    const result = assignClientContext(mockContext);
+
+    // Check each property individually
+    expect(result.frontmatter).toEqual(mockContext.frontmatter);
+    expect(result.pathname).toEqual(mockContext.pathname);
+    expect(result.headings).toEqual(mockContext.headings);
+    expect(result.readingTime).toEqual(mockContext.readingTime);
+    expect(result.filename).toEqual(mockContext.filename);
+
+    // Alternatively, you can check the entire object at once
```

Comment:
What are these comments?

Reply:
As indicated in the description, I still need to do a self-review. And these are comments I took during the "creation" of the test they must be removed because they just represent my thoughts 😅